### PR TITLE
support MathJax

### PR DESCRIPTION
**Pull request by lizebang**

### CODE REVIEW

File: layouts/partials/footer.html [hunk 1]
```diff
@@ -1,5 +1,37 @@
...
+			}
+		);
+		MathJax.Hub.Queue(function() {
+			var i, text, code, codes = document.getElementsByTagName('code');
+			for (i = 0; i < codes.length;) {
+				code = codes[i];
+				if (code.parentNode.tagName !== 'PRE' &&
+					code.childElementCount === 0) {
+					text = code.textContent;
+						if (/^\\\((.|\s)+\\\)$/.test(text) ||
```
Comment (kakawait) [inline layouts/partials/footer.html]: Could you please explain me why did you perform such regex?

Comment (AbangSD) [reply layouts/partials/footer.html]: The effect is to find out all the code labels, one by one check. If the text inside the text looks like a mathematical formula, put the code label removed. Of course, there are other restrictions, such as not processing <pre> tag code, handling code labels and do not contain other label.

Comment (AbangSD) [reply layouts/partials/footer.html]: The effect is to find out all the code labels, one by one check. If the text inside the text looks like a mathematical formula, put the code label removed. Of course, there are other restrictions, such as not processing *<pre>* tag code, handling code labels and do not contain other label.

Comment (AbangSD) [reply layouts/partials/footer.html]: The effect is to find out all the code labels, one by one check. If the text inside the text looks like a mathematical formula, put the code label removed. Of course, there are other restrictions, such as not processing *pre* tag code, handling code labels and do not contain other label.

**Comment by kakawait** [comment]
Thank but I'll close it in flavor of #294 for that first version.

**Pull request closed without merging**